python/tank/util/defaults_manager.py

```diff
@@ -60,7 +60,7 @@ def get_http_proxy(self):
                   None if not necessary.
         """
         from . import shotgun
-        return shotgun.get_associated_sg_config_data().get("http_proxy")
+        return shotgun.get_associated_sg_config_data().get("http_proxy") or super(CoreDefaultsManager, self).get_http_proxy()
```

Comment:
is this the right order of precedence? if you have a home computer with a special proxy, maybe it makes more sense to want to use the defaults mgr first and if that lookup fails fall back on the project settings?